oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+    --display-name <image-name> \
+    --launch-mode <launch-mode> \
+    --image-source-object-name <object-name> \
+    --bucket-name <bucket-name> \
+    --operating-system "Ubuntu" \
+    --operating-system-version <ubuntu-version-number> \
+    --source-image-type QCOW2 \
+    --wait-for-state AVAILABLE
+
+
+Create OKE nodes with Ubuntu Images
+-------------------------------------
+
+The following steps on creating nodes assumes you have an existing OKE cluster on Oracle Cloud, but it is not required to have existing nodes. If you don't have an OKE cluster prepared then Oracle's documentation for `creating a cluster`_ is a good place to start.
+
+Create managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Managed nodes are node instances whose lifecycle is managed by the OKE service. Since this is a Limited Availability release of Ubuntu images for OKE, you can only create managed nodes through the Oracle Cloud API (``oci`` CLI or SDK). The ability to create managed nodes from the Oracle Cloud UI will be added later.
+
+To create a managed node, start by copying the following cloud-init script into a file called ``user-data.yaml``.
+
+.. code:: yaml
+
+   #cloud-config
+   
+   runcmd:
+     - oke bootstrap
+
+Then, create a placement configuration file to specify where in Oracle Cloud the Managed node pool should be created and save the file as ``placement-config.json``.
+
+.. code:: json 
+
+   [{
+     "compartmentId":"<compartment-id>",
+     "availabilityDomain":"<availability-domain>",
+     "subnetId":"<subnet-id>"
+   }]
+
+
+Lastly, replace the values and run the following command to create the Managed node pool:
+
+.. code:: bash
+   
+  oci ce node-pool create \
+    --cluster-id=<cluster-id> \
+    --compartment-id=<compartment-id> \
+    --name=<pool-name> \
+    --node-shape=<node-shape> \
+    --size=<pool-count> \
+    --kubernetes-version="1.29.1" \
+    --node-image-id=<ubuntu-image-id> \
+    --placement-configs="$(cat placement-config.json)" \
+    --node-metadata='{"user_data": "'"$(base64 user-data.yaml)"'"}'
+
+
+View the node pool status in Oracle Cloud by navigating to :guilabel:`Kubernetes Clusters (OKE)` and click on your cluster, then select :guilabel:`Resources` > :guilabel:`Node pools` and click on the latest node pool.
+
+Everything will be running as expected when the :guilabel:`Kubernetes node condition` and :guilabel:`Node state` of all the nodes are labelled :guilabel:`Ready`.
+
+Create self-managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following instructions assume that you have configured your OKE cluster to work with self-managed nodes. If you have not done this, refer to the Oracle documentation for `working with self-managed nodes`_
+
+Before adding a self-managed node, ensure you have configured ``kubectl`` to your OKE cluster with the following command. This process will be easier if ``kubectl`` is configured for a single OKE cluster.
```

Comment:
should this be:
```suggestion
Before adding a self-managed node, ensure you have configured ``kubectl`` on your OKE cluster with the following command. This process will be easier if ``kubectl`` is configured for a single OKE cluster.
```

Reply:
Personally, I think it makes more sense with 'for'. ``kubectl`` is a client that talks to your cluster(s), the OKE service itself doesn't have any configuration to use it.